examples/broker.conf.example

```diff
@@ -0,0 +1,30 @@
+##--------------------------------------------------------------------
+## Broker
+##
+## Message broker options
+##--------------------------------------------------------------------
+## Note: This is an example of how to configure this feature
+##       you should copy and paste the below data into the emqx.conf for working
+
+## Note: If you and finding how to configure EMQX itself, have a look at the node.conf.example
+
+broker {
+    ## Session locking strategy in a cluster
+    ## Type:
+    ##   - local :: only lock the session on the current node
+    ##   - leader :: select only one remote node to lock the session
+    ##   - quorum :: select some nodes to lock the session
+    ##   - all :: lock the session on all the nodes in the cluster
+    session_locking_strategy = quorum
+
```

Comment:
the entire broker is hidden, and move shared_subscription_strategy to `mqtt` root key.